client/src/components/CHeader.style.js

```diff
@@ -0,0 +1,43 @@
+import styled, { css } from 'styled-components';
+
+export const Header = styled.header`
```

Comment:
can header and headerwrapper be consolidated into one style?

Reply:
I needed the two one as a container and the second put padding in it